### PR TITLE
Podcast episode

### PR DESCRIPTION
**Pull request by mre**

### CODE REVIEW

**Comment by mre** [comment]
@prscheduler 28/11/2024T16:00

File: content/podcast/s03e04-gitbutler/index.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+### Links From The Episode (In Chronological Order)
+
+- [sturdy](https://getsturdy.com/) - Kiril's earlier excursion into version control
+- [Why GitHub Actually Won](https://blog.gitbutler.com/why-github-actually-won/) - Comparing git to other version control software
+- [git2-rs](https://github.com/rust-lang/git2-rs) - Rust bindings for libgit2
+- [gitoxide](https://github.com/Byron/gitoxide) - Pure Rust implementation of git
+- [jujutsu](https://github.com/martinvonz/jj) - Alternative frontend to git repositories
+- [jgit](https://www.eclipse.org/jgit/) - Java implementation of git
+- [Git Book - Plumbing and Porcelain](https://git-scm.com/book/en/v2/Git-Internals-Plumbing-and-Porcelain) - Explaining "git porcelain", written by Scott
```
Comment (M3t0r) [inline content/podcast/s03e04-gitbutler/index.md]:
```suggestion
```

I cut the section that mentioned this.

File: content/podcast/s03e04-gitbutler/index.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+- [Slint](https://slint-ui.com/) - Declarative Rust GUI toolkit
+- [Svelte](https://svelte.dev/) - JS frontend framework used by GitButler
+- [Zed](https://zed.dev/) - An editor with their very own UI toolkit
+- [Rust in Production: Zed](/podcast/s03e01-zed) - Podcast episode with Conrad Irwin from Zed
+- [Atom](https://github.com/atom/atom) - The hackable text editor
+- [Tauri IPC Guide](https://tauri.app/concept/inter-process-communication/) - How Tauri handles messages between front and backend
+- [Clippy](https://github.com/rust-lang/rust-clippy) - Rust's official linter
+- [Simple Made Easy](https://www.infoq.com/presentations/Simple-Made-Easy/) - Rich Hickey's talk that Kiril watched more than 10 times
+- [Labradoodle](https://en.wikipedia.org/wiki/Labradoodle) - A dog breed
+- [I Will Survive](https://en.wikipedia.org/wiki/I_Will_Survive) - Gloria Gaynor's iconic song: "At first I was afraid, I was petrified"
```
Comment (M3t0r) [inline content/podcast/s03e04-gitbutler/index.md]: I think the description explains too much :grinning: Also, I think the YouTube link would be more helpful to listeners.

File: content/podcast/s03e04-gitbutler/index.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+- [Dioxus](https://dioxuslabs.com/) - Rust GUI toolkit for web, desktop, mobile, and more
+- [Slint](https://slint-ui.com/) - Declarative Rust GUI toolkit
+- [Svelte](https://svelte.dev/) - JS frontend framework used by GitButler
+- [Zed](https://zed.dev/) - An editor with their very own UI toolkit
+- [Rust in Production: Zed](/podcast/s03e01-zed) - Podcast episode with Conrad Irwin from Zed
+- [Atom](https://github.com/atom/atom) - The hackable text editor
+- [Tauri IPC Guide](https://tauri.app/concept/inter-process-communication/) - How Tauri handles messages between front and backend
+- [Clippy](https://github.com/rust-lang/rust-clippy) - Rust's official linter
+- [Simple Made Easy](https://www.infoq.com/presentations/Simple-Made-Easy/) - Rich Hickey's talk that Kiril watched more than 10 times
+- [Labradoodle](https://en.wikipedia.org/wiki/Labradoodle) - A dog breed
```
Comment (M3t0r) [inline content/podcast/s03e04-gitbutler/index.md]:
```suggestion
- [Labradoor](https://en.wikipedia.org/wiki/Labrador_Retriever) - A magical dog breed
```

I actually wrote down the wrong breed :facepalm:

File: content/podcast/s03e04-gitbutler/index.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+draft = false
+aliases = ["/p/s03e04"]
+[extra]
+guest = "Scott Chacon and Kiril Videlov"
+role = "Co-Founders"
+season = "03"
+episode = "04"
+series = "Podcast"
++++
+
```
Comment (M3t0r) [inline content/podcast/s03e04-gitbutler/index.md:14]:
```suggestion

<div><script id="letscast-player-4553b0d1" src="https://letscast.fm/podcasts/rust-in-production-82281512/episodes/gitbutler-with-scott-chacon-and-kiril-videlov/player.js?size=s"></script></div>

```

**Review by M3t0r — CHANGES_REQUESTED** [review]

File: content/podcast/s03e04-gitbutler/index.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+- [Dioxus](https://dioxuslabs.com/) - Rust GUI toolkit for web, desktop, mobile, and more
+- [Slint](https://slint-ui.com/) - Declarative Rust GUI toolkit
+- [Svelte](https://svelte.dev/) - JS frontend framework used by GitButler
+- [Zed](https://zed.dev/) - An editor with their very own UI toolkit
+- [Rust in Production: Zed](/podcast/s03e01-zed) - Podcast episode with Conrad Irwin from Zed
+- [Atom](https://github.com/atom/atom) - The hackable text editor
+- [Tauri IPC Guide](https://tauri.app/concept/inter-process-communication/) - How Tauri handles messages between front and backend
+- [Clippy](https://github.com/rust-lang/rust-clippy) - Rust's official linter
+- [Simple Made Easy](https://www.infoq.com/presentations/Simple-Made-Easy/) - Rich Hickey's talk that Kiril watched more than 10 times
+- [Labradoor](https://en.wikipedia.org/wiki/Labrador_Retriever) - A magical dog breed
```
Comment (M3t0r) [inline content/podcast/s03e04-gitbutler/index.md]:
```suggestion
- [Labrador](https://en.wikipedia.org/wiki/Labrador_Retriever) - A magical dog breed
```

It's right there in the link, and I still fucked up :-D

**Review by M3t0r — CHANGES_REQUESTED** [review]

File: content/podcast/s03e04-gitbutler/index.md (hunk 2 above)
Comment (mre) [reply content/podcast/s03e04-gitbutler/index.md]: true, changed

**Comment by mre** [comment]
@prscheduler 28/11/2024T16:00

**Pull request merged**